src/legacy/core_plugins/tile_map/common/ems_client.js

```diff
@@ -111,8 +114,10 @@ export class EMSClient {
     this._emsLandingPageUrl = typeof landingPageUrl === 'string' ? landingPageUrl : '';
     this._language = typeof language === 'string' ? language : DEFAULT_LANGUAGE;
 
-    this._invalidateSettings();
+    this._fetchFunction = typeof fetchFunction === 'function' ? fetchFunction : fetch;
+    this._proxyPath = typeof proxyPath === 'string' ? proxyPath : '';
```

Comment:
this is required because we have no guarantees on how kibana is deployed (e.g. if it sits behind a proxy). this allows the client of `EMSClient` to prepend a path to the kibana deployment location